feature/eks-cluster-creation/aws-resource-groups.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_resourcegroups_group" "eks_resource_group" {
+  name = var.resource_group_name
+
+  resource_query {
+    query = <<JSON
+{
+  "ResourceTypeFilters": [
+    "AWS::EC2::Instance"
+  ],
+  "TagFilters": [
+    {
+      "Key": "Environment",
+      "Values": ["${var.environment}"]
+    }
+  ]
+}
+JSON
```

Comment:
I know there is this construct, but isn't there some better way now to perform it in terraform? Just curious.

Reply:
Sadly, I don't see any other option for now, its the official way: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/resourcegroups_group